projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts

```diff
@@ -171,13 +173,13 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
            */
           // Prevent dropping a parent below another parent (only sort
           // nodes under the same parent)
-/*           if(node.parent !== data.otherNode.parent){
-            return false;
-          }
-          // Don't allow dropping *over* a node (would create a child)
-          return ["before", "after"];
-*/
-           return true;
+          /*           if(node.parent !== data.otherNode.parent){
+                      return false;
+                    }
+                    // Don't allow dropping *over* a node (would create a child)
+                    return ["before", "after"];
+          */
+          return true;
```

Comment:
remove these commented code if not useful.

Reply:
Ok, I Will check and remove if not useful